ext/standard/exec.c

```diff
@@ -272,8 +269,7 @@ PHP_FUNCTION(passthru)
    Escape all chars that could possibly be used to
    break out of a shell command
 
-   This function emalloc's a string and returns the pointer.
-   Remember to efree it when done with it.
+   This function allocates a new zend_string, remember to free it when done.
```

Comment:
Not necessarily a new string as it can also be `ZSTR_EMPTY_ALLOC()`.
I would also replace "free" with "release" because we have both zend_string_(e)free and zend_string_release

Reply:
Done